Swift.gitignore

```diff
@@ -46,7 +46,7 @@ playground.xcworkspace
 # you should judge for yourself, the pros and cons are mentioned at:
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
-# Pods/
+Pods/
```

Comment:
Please read the comment above this rule, as the recommendation for Cocoapods is to not add this directory to source control.

Without more information and discussion about whether that recommendation has changed, I'd like to keep this as-is.